instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -95,6 +102,11 @@ def process_request(self, request):
         tracer = get_tracer(__name__, __version__)
 
         attributes = collect_request_attributes(environ)
+        for attr in self._traced_request_attrs:
+            if hasattr(request, attr):
```

Comment:
It would be nice to support nested attributes for GET and POST parameters e.g. `OTEL_PYTHON_DJANGO_TRACED_REQUEST_ATTRS="GET.some_query_variable"`

If this PR is just to be compatible with OpenTracing, feel free to disregard

Reply:
Yea, would also be nice to be able to extract HTTP headers and probably we can add support for that later. For now I just wanted to clear the migration path for opentracing users.